src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion_adapter.py

```diff
@@ -0,0 +1,798 @@
+# Copyright 2022 The HuggingFace Team. All rights reserved.
```

Comment:
Do we need to attribute the original authors?

Reply:
Yes good idea

Reply:
done!